content/integrations/agents/LINUX-agent/LINUX-standard-install.md

```diff
@@ -8,9 +8,31 @@ author: Lawrence Lane
 #pre: "<i class='fa fa-download'></i> &nbsp;"
 weight: 1
 ---
-This integration’s package (computed metrics, dashboards, and policies that will give you important events and alerts) will be automatically enabled and provisioned to your account as soon as CloudWisdom receives data from the integration. The PACKAGES button on the integration setup page will become active once data is received, so you’ll be able to disable and re-enable the package at will.
+This integration’s package (computed metrics, dashboards, and policies) is automatically enabled and provisioned to your account as soon as CloudWisdom receives data.
 
-## 1. Copy Install Command From Linux Integration Setup Page
+## Cost Optimization Quick Setup
```

Comment:
I think it makes more sense for this section to be after the original Standard Install section. It's a bit confusing to click on Standard Install and then see this as the first section.